scripting/include/srccoop/manager.inc

```diff
@@ -515,49 +504,66 @@ methodmap CoopManager
 	
 }
 
-public Action StartTimerCallback(Handle pTimer)
+// Runs every 1 second.
+// 
+public Action Timer_DeincrementUntilMapStart(Handle hTimer)
 {
-	if (!--data.m_iSecondsToStart)
-	{
-		data.m_pStartTimer = null;
-		CoopManager.Start();
-		return Plugin_Stop;
-	}
-	
-	if (data.m_iSecondsToStart == 3)
-	{
-		EmitSoundToAll(Conf.SND_3, .volume = Conf.SND_TICK_VOL);
-	}
-	else if (data.m_iSecondsToStart == 2)
-	{
-		EmitSoundToAll(Conf.SND_2, .volume = Conf.SND_TICK_VOL);
-	}
-	else if (data.m_iSecondsToStart == 1)
+	--data.m_iSecondsToStart;
+	if (data.m_iSecondsToStart <= 0)
```

Comment:
These have been safe to combine in SP so far

Reply:
Keeping this for readability. I typically seperate pre/post-increment operations into their own lines.